.gitignore

```diff
@@ -1,2 +1,3 @@
 water.test
 water.test.exe
+.idea
```

Comment:
I'd prefer not adding IDE specific stuff here. I think those are better to live in `.gitignore_global`.